src/test/tests/dependencyUtils.test.ts

```diff
@@ -0,0 +1,227 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+import { IGraphResponse } from 'jfrog-client-js';
+import { describe } from 'mocha';
+import { DependencyScanResults, ScanResults } from '../../main/types/workspaceIssuesDetails';
+import { DependencyUtils } from '../../main/treeDataProviders/utils/dependencyUtils';
+import { DescriptorTreeNode } from '../../main/treeDataProviders/issuesTree/descriptorTree/descriptorTreeNode';
+import { assert } from 'chai';
+import { IImpactGraph } from 'jfrog-ide-webview';
+import { BuildTreeErrorType, RootNode } from '../../main/treeDataProviders/dependenciesTree/dependenciesRoot/rootTree';
+import { FileScanBundle, FileScanError, NotEntitledError, ScanCancellationError } from '../../main/utils/scanUtils';
+import { FileTreeNode } from '../../main/treeDataProviders/issuesTree/fileTreeNode';
+import { LogManager } from '../../main/log/logManager';
+import { PackageType } from '../../main/types/projectType';
+import { createDependency, createRootTestNode } from './utils/treeNodeUtils.test';
+import { DependenciesTreeNode } from '../../main/treeDataProviders/dependenciesTree/dependenciesTreeNode';
+import { DependencyIssuesTreeNode } from '../../main/treeDataProviders/issuesTree/descriptorTree/dependencyIssuesTreeNode';
+import { IssueTreeNode } from '../../main/treeDataProviders/issuesTree/issueTreeNode';
+
+describe('Dependency Utils Tests', () => {
```

Comment:
I'm not sure the tests are running
https://github.com/jfrog/jfrog-vscode-extension/actions/runs/4366198697/jobs/7635874087